routers/rooms.js

```diff
@@ -499,6 +553,211 @@ router.post('/room/:id/content_flags', authenticateToken, requiresRoomPermission
     }
 });
 
+router.post('/room/:id/moderation-suspend', authenticateDeveloperToken, async (req, res) => {
```

Comment:
* https://api.compensationvr.tk/api/rooms/room/:id/moderation-suspend
* Suspends the specified room, with a note.
* Usable by developers only.